x-pack/plugins/enterprise_search/public/applications/app_search/components/curations/views/curations.tsx

```diff
@@ -0,0 +1,178 @@
+      truncateText: true,
+      mobileOptions: {
+        header: true,
+        // Note: the below props are valid props per https://elastic.github.io/eui/#/tabular-content/tables (Responsive tables), but EUI's types have a bug reporting it as an error
+        // @ts-ignore
+        enlarge: true,
+        width: '100%',
+        truncateText: false,
+      },
+    },
+    {
+      field: 'last_updated',
+      name: i18n.translate(
+        'xpack.enterpriseSearch.appSearch.engine.curations.table.column.lastUpdated',
+        { defaultMessage: 'Last updated' }
+      ),
+      width: '30%',
+      dataType: 'string',
+    },
+    {
+      width: '120px',
+      actions: [
+        {
+          name: i18n.translate(
+            'xpack.enterpriseSearch.appSearch.engine.curations.table.editAction',
+            { defaultMessage: 'Edit' }
+          ),
+          description: i18n.translate(
+            'xpack.enterpriseSearch.appSearch.engine.curations.table.editAction',
+            { defaultMessage: 'Edit' }
+          ),
+          type: 'icon',
+          icon: 'pencil',
+          color: 'primary',
+          onClick: (curation: Curation) => {
+            const { navigateToUrl } = KibanaLogic.values;
+            const url = generateEnginePath(ENGINE_CURATION_PATH, { curationId: curation.id });
+            navigateToUrl(url);
+          },
+          'data-test-subj': 'CurationsTableEditButton',
+        },
+        {
+          name: i18n.translate(
+            'xpack.enterpriseSearch.appSearch.engine.curations.table.deleteAction',
+            { defaultMessage: 'Delete' }
+          ),
+          description: i18n.translate(
+            'xpack.enterpriseSearch.appSearch.engine.curations.table.deleteAction',
+            { defaultMessage: 'Delete' }
+          ),
+          type: 'icon',
+          icon: 'trash',
+          color: 'danger',
+          onClick: (curation: Curation) => deleteCurationSet(curation.id),
+          'data-test-subj': 'CurationsTableDeleteButton',
+        },
+      ],
+    },
+  ];
+
+  return (
+    <EuiBasicTable
+      columns={columns}
+      items={curations}
+      responsive
+      hasActions
+      loading={dataLoading}
+      noItemsMessage={
+        <EuiEmptyPrompt
+          iconType="pin"
+          title={
+            <h4>
+              {i18n.translate(
+                'xpack.enterpriseSearch.appSearch.engine.curations.table.empty.noCurationsTitle',
+                { defaultMessage: 'No curations yet' }
+              )}
+            </h4>
+          }
+        />
+      }
+      pagination={{
+        pageIndex: meta.page.current - 1,
+        pageSize: meta.page.size,
+        totalItemCount: meta.page.total_results,
+        hidePerPageOptions: true,
+      }}
+      onChange={({ page }: { page: { index: number } }) => {
+        const { index } = page;
+        onPaginate(index + 1); // Note on paging - App Search's API pages start at 1, EuiBasicTables' pages start at 0
+      }}
```

Comment:
We repeat this logic & comment in a few (2-3) other places in App Search. I plan on adding a new PR right after this one that DRY's oiut both handling our `meta.page` snake_case API vars to camelCase EUI props conversions, as well as automatically handling this off-by-one calculations for us (so we hopefully don't forget this in the future).